common/daq/README.md

```diff
@@ -0,0 +1,20 @@
+# DAQ Code Generation
```

Comment:
Might be good to add a pipenv or requirements.txt for the python packages needed to run the generator